### PR TITLE
make webkitgtk@2.4.11 available as a bottle

### PR DESCRIPTION
**Pull request by c-alpha**
Simply takes too long to compile...

### CODE REVIEW

**Comment by c-alpha** [comment]
@yeled ? @gauteh ? @cpiro ?

**Comment by yeled** [comment]
super! trying it now

File: Formula/webkitgtk@2.4.11.rb [hunk 1]
```diff
@@ -4,6 +4,16 @@ class WebkitgtkAT2411 < Formula
   url "https://webkitgtk.org/releases/webkitgtk-2.4.11.tar.xz"
   sha256 "588aea051bfbacced27fdfe0335a957dca839ebe36aa548df39c7bbafdb65bf7"
 
+  bottle do
+    # NOTE: when you merge this PR, be sure to update the the URL below to:
+    #       https://github.com/astroidmail/homebrew-astroid/raw/master/bottles
+    #       (and to remove this note, of course)
+    #
+    root_url "https://github.com/c-alpha/homebrew-astroid/raw/webkitgtk%402.4.11-bottle/bottles"
+    rebuild 1
```
Comment (yeled) [inline Formula/webkitgtk@2.4.11.rb]: This isn't necessary. Note the "`.1`" that homebrew adds:

```
12:30 mp01:~/src/homebrew-astroid[] % wget https://github.com/c-alpha/homebrew-astroid/raw/webkitgtk%402.4.11-bottle/bottles/webkitgtk@2.4.11-2.4.11.high_sierra.bottle.1.tar.gz
--2018-02-07 12:30:44--  https://github.com/c-alpha/homebrew-astroid/raw/webkitgtk%402.4.11-bottle/bottles/webkitgtk@2.4.11-2.4.11.high_sierra.bottle.1.tar.gz
Resolving github.com (github.com)... 18.195.85.27
Connecting to github.com (github.com)|18.195.85.27|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2018-02-07 12:30:44 ERROR 404: Not Found.

12:30 mp01:~/src/homebrew-astroid[] % wget https://github.com/c-alpha/homebrew-astroid/raw/webkitgtk%402.4.11-bottle/bottles/webkitgtk@2.4.11-2.4.11.high_sierra.bottle.tar.gz
--2018-02-07 12:30:48--  https://github.com/c-alpha/homebrew-astroid/raw/webkitgtk%402.4.11-bottle/bottles/webkitgtk@2.4.11-2.4.11.high_sierra.bottle.tar.gz
Resolving github.com (github.com)... 18.195.85.27
Connecting to github.com (github.com)|18.195.85.27|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://raw.githubusercontent.com/c-alpha/homebrew-astroid/webkitgtk%402.4.11-bottle/bottles/webkitgtk%402.4.11-2.4.11.high_sierra.bottle.tar.gz [following]
--2018-02-07 12:30:49--  https://raw.githubusercontent.com/c-alpha/homebrew-astroid/webkitgtk%402.4.11-bottle/bottles/webkitgtk%402.4.11-2.4.11.high_sierra.bottle.tar.gz
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.16.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.16.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 9963360 (9.5M) [application/octet-stream]
Saving to: ‘webkitgtk@2.4.11-2.4.11.high_sierra.bottle.tar.gz’
```

Comment (c-alpha) [reply Formula/webkitgtk@2.4.11.rb]: Well spotted, thanks! Removed.

**Comment by yeled** [comment]
This works! Hurray!

I will update the formula with your patches.

**Comment by yeled** [comment]
Do you want to update the URL in your branch here and I will just merge it?

**Comment by c-alpha** [comment]
> Do you want to update the URL in your branch here and I will just merge it?

Done.

**Comment by gauteh** [comment]
Does the README need to be updated for this?

**Pull request merged**